### ISSUE TITLE
CocoaLibEvent does not contain bitcode

### ISSUE THREAD

**Issue by olafurnsig** [react-native-community/react-native-tvos#70]
After fresh setup according to docs, tvOS build encounters error:

`ios/Pods/CocoaLibEvent/lib_tvos/libevent.a(event.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64`

CocoaLibEvent is something that Flipper seems to require, and I can't find any proper info on how to enable bitcode for CocoaLibEvent pod.

If I comment out the references to everything related to Flipper in the Podfile, and comment out everything related to Flipper, the app runs.

## React Native version:
info Fetching system and libraries information...
System:
    OS: macOS 10.15.5
    CPU: (16) x64 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
    Memory: 1.80 GB / 32.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 10.17.0 - /usr/local/opt/node@10/bin/node
    Yarn: 1.22.4 - ~/.yarn/bin/yarn
    npm: 6.11.3 - /usr/local/opt/node@10/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.9.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 13.5, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK:
      API Levels: 23, 24, 26, 28, 29
      Build Tools: 28.0.3, 29.0.0
      System Images: android-27 | Android TV Intel x86 Atom, android-28 | Android TV Intel x86 Atom, android-29 | Google APIs Intel x86 Atom, android-Q | Android TV Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6241897
    Xcode: 11.5/11E608c - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_131 - /usr/bin/javac
    Python: 2.7.13 - /Library/Frameworks/Python.framework/Versions/2.7/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.11.0 => 16.11.0 
    react-native: Not Found
    react-native-tvos:  0.62.2-0 

## Steps To Reproduce

1. npx react-native init MyApp --template=react-native-tvos@latest
2. Open Xcode to run on connected AppleTV 4K

Describe what you expected to happen:

The thing actually runs without adjustments, or at least a mention of this gotcha in the readme :)

**Comment by dlowder-salesforce** [comment]
I'll look into this.... I didn't hit this issue on a non-4k Apple TV or in the simulator

**Comment by dlowder-salesforce** [comment]
Ok I can now reproduce the issue -- investigating.

Just so you know, Flipper does not currently support connections to physical devices, so even when I fix the compilation issue, Flipper may not actually work on a real Apple TV.

https://fbflipper.com/docs/getting-started/ios-native

**Comment by olafurnsig** [comment]
Thanks for looking into this!

Yeah I actually have no intensions of using Flipper, just bugged me to not be able to build from the get-go.

I just disabled everything regarding Flipper to get up and running.

**Comment by dlowder-salesforce** [comment]
@olafurnsig  A quicker way to get up and running on a device is to add a user defined setting to your project build settings:

ENABLE_BITCODE=NO

However, not doing the device build with Bitcode enabled might cause problems with submission to the App Store, so I'm going to have to think carefully about whether I should put this into the new project template.

**Comment by dlowder-salesforce** [comment]
@olafurnsig this is fixed in the 0.62.2 maintenance branch, and will be in the next release. I disabled Bitcode for debug device builds (Flipper is only supported for debug builds anyway).

If you are using the community CLI (`@react-native-community/cli`), you can try this by doing

```
react-native init TestApp --template=https://github.com/react-native-community/react-native-tvos#tvos-v0.62.2
```

**Comment by olafurnsig** [comment]
> @olafurnsig this is fixed in the 0.62.2 maintenance branch, and will be in the next release. I disabled Bitcode for debug device builds (Flipper is only supported for debug builds anyway).
> 
> If you are using the community CLI (`@react-native-community/cli`), you can try this by doing
> 
> ```
> react-native init TestApp --template=https://github.com/react-native-community/react-native-tvos#tvos-v0.62.2
> ```

Thanks for the hard work Doug!

But yeah, Bitcode is a requisite for a AppleTV release in the appstore, so it should only be disabled for debug builds.
